### DISCUSSION TITLE
你好！请问怎样在网络离线状态下，打开MouseInc设置，谢谢！

### DISCUSSION THREAD

**Discussion by Chiltonn** [shuax/MouseInc.Settings#305] [General]
你好！请问怎样在网络离线状态下，打开MouseInc设置，谢谢！

软件版本：2.12.1 B
已安装：WebView2 V1.3.153.53 官方安装版

**Comment by fodereas** [comment]
https://github.com/shuax/MouseInc.Settings/discussions/257
这里的截图很清楚了
zip不解压 直接放到文件夹里就行

**Reply by kevzcr** [reply]
原来截图说得好清楚，眼睛楞是没看出来。
